Samples/Islands/cpp-win32-unpackaged/SimpleIslandApp.cpp

```diff
@@ -192,13 +192,16 @@ void MyRegisterClass(HINSTANCE hInstance, const wchar_t* szWindowClass)
 //
 HWND InitInstance(HINSTANCE /*hInstance*/, int nCmdShow, const wchar_t* szTitle, const wchar_t* szWindowClass)
 {
-   HWND hWnd = CreateWindowW(szWindowClass, szTitle, WS_OVERLAPPEDWINDOW,
-      CW_USEDEFAULT, 0, CW_USEDEFAULT, 0, nullptr, nullptr, ::GetModuleHandle(NULL), nullptr);
-   winrt::check_bool(hWnd != NULL);
-
-   ShowWindow(hWnd, nCmdShow);
-   UpdateWindow(hWnd);
-   return hWnd;
+    HWND hWnd = CreateWindowW(szWindowClass, szTitle, WS_OVERLAPPEDWINDOW,
+       CW_USEDEFAULT, 0, CW_USEDEFAULT, 0, nullptr, nullptr, ::GetModuleHandle(NULL), nullptr);
+    winrt::check_bool(hWnd != NULL);
+    if(!hWnd)
```

Comment:
Have you considered "if (hwnd != INVALID_HANDLE_VALUE)" ?

Reply:
I found no usage of INVALID_HANDLE_VALUE in the Samples repo. In other places, they are doing this check using NULL